lib/bugsnex/api.ex

```diff
@@ -15,6 +15,11 @@ defmodule Bugsnex.Api do
     post("/", body)
   end
 
+  def send_deploy(deploy) do
+    body = Poison.encode!(deploy)
+    post("/deploy", body)
```

Comment:
ok, that wasn't expanded and i was too lazy to read the whole file. sorry 'bout that